### PR TITLE
Allow Skipping of Invalid Activity Functions in Struct

### PR DESCRIPTION
**Pull request by shannontan-bolt**
This allows activities to be registered that may have some Temporal activities along with other functions.

I couldn't find a way to test the negative case without causing a failure in the test start code that would require a re-run.

Intended to address #356.

### CODE REVIEW

**Review by vitarb — APPROVED** [review]

**Review by vitarb — CHANGES_REQUESTED** [review]
Please address build failure.

File: internal/internal_worker_test.go [hunk 1]
```diff
@@ -1726,6 +1740,12 @@ func testActivityReturnStructPtrPtr() (**testActivityResult, error) {
 	return &r, nil
 }
 
+type testActivityStructWithFns struct{}
+
+func (t *testActivityStructWithFns) ValidActivity(ctx context.Context) error { return nil }
+
+func (t *testActivityStructWithFns) InvalidActivity(ctx context.Context) { return }
```
Comment (vitarb) [inline internal/internal_worker_test.go:1747]: This `return` should be removed, as the code won't compile this way.

**Comment by Sushisource** [comment]
@shannontan-bolt I've fixed the problems and made a new PR here. I'm closing this one in favor of that. Thanks!

**Comment by shannontan-bolt** [comment]
Thanks!

**Pull request closed without merging**